Assets/Treasure/TDK/Runtime/Infrastructure/TDKTimeKeeper.cs

```diff
@@ -29,6 +29,9 @@ private IEnumerator GetServerTime()
 
             using (UnityWebRequest webRequest = UnityWebRequest.Get(endpoint))
             {
+                // Set the API key header
+                webRequest.SetRequestHeader("x-api-key", TDK.Instance.AppConfig.ApiKey);
```

Comment:
Is it needed for the time keeper? Even if its a public endpoint

Reply:
not for the time keeper. just the /ingress apis